apis/migrations.js

```diff
@@ -0,0 +1,26 @@
+/* eslint no-await-in-loop: 0 */
+import Sentry from 'sentry-expo';
+import { executeTransaction } from './database';
+
+const migrations = [
+  () =>
+    executeTransaction(
+      'CREATE TABLE IF NOT EXISTS locations (id integer PRIMARY KEY NOT NULL, key text, coordinateKey text, createdAt text, resources text, status text, uploaded int)'
+    ),
+  () => executeTransaction('ALTER TABLE locations ADD COLUMN version integer NOT NULL DEFAULT 0'),
+];
+
+export default async function createDatabases() {
```

Comment:
Will this get run every time the app starts up? Should we maybe add a check in to see if the DB is already up to date before running these migrations?

Reply:
This method will be run Everytime, but internally it checks the version number it has and the version it requires. Does that help clarify?

Reply:
I had a feeling it would but just wanted to check! 👍